packages/components/src/internal/components/domainproperties/models.tsx

```diff
@@ -1695,6 +1698,7 @@ export interface IDomainFormDisplayOptions {
     hideImportExport?: boolean;
     hideConditionalFormatting?: boolean;
     hideInferFromFile?: boolean;
+    showScannableOption?: boolean;
```

Comment:
not sure where to put this comment, but I pulled up the PR quick locally and noticed that the new scannable property isn't shown in the grid when you toggle the field editor from Detail mode -> Summary mode.

Reply:
Add check for SampleSet and app, and adjusted sort location